include/bluetooth/mesh/gen_lvl.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ * @file
+ * @defgroup bt_mesh_gen_lvl Bluetooth Mesh Generic Level models
+ * @{
+ * @brief Common API for the Bluetooth Mesh Generic Level models.
+ */
+#ifndef BT_MESH_GEN_LVL_H__
+#define BT_MESH_GEN_LVL_H__
+
+#include <bluetooth/mesh.h>
+#include <bluetooth/mesh/model_types.h>
+
+/** @cond INTERNAL_HIDDEN  */
+#define GEN_LVL_OP_GET			BT_MESH_MODEL_OP_2(0x82, 0x05)
+#define GEN_LVL_OP_SET			BT_MESH_MODEL_OP_2(0x82, 0x06)
+#define GEN_LVL_OP_SET_UNACK		BT_MESH_MODEL_OP_2(0x82, 0x07)
+#define GEN_LVL_OP_STATUS		BT_MESH_MODEL_OP_2(0x82, 0x08)
+#define GEN_LVL_OP_DELTA_SET		BT_MESH_MODEL_OP_2(0x82, 0x09)
+#define GEN_LVL_OP_DELTA_SET_UNACK	BT_MESH_MODEL_OP_2(0x82, 0x0A)
+#define GEN_LVL_OP_MOVE_SET		BT_MESH_MODEL_OP_2(0x82, 0x0B)
+#define GEN_LVL_OP_MOVE_SET_UNACK	BT_MESH_MODEL_OP_2(0x82, 0x0C)
+/** @endcond */
+
+/** Generic Level set message parameters.  */
+struct gen_lvl_set {
+	/** New level. */
+	s16_t level;
```

Comment:
May I provide a counter-argument here? This is a structure representing a message field, so it is better to retain it as named in the spec (since that is how all other structure fields are named). As I see, `lvl` is only used for API names to avoid long names.

Reply:
resolved.